helmfile/charts/gatekeeper/podsecuritypolicies/templates/custom/mutations/read-only-root.yaml

```diff
@@ -0,0 +1,87 @@
+{{- range $namespace, $services := .Values.constraints }}
+{{- range $name, $value := $services }}
+{{- if get $value.allow "readOnlyRootFilesystem" }}
+---
+apiVersion: mutations.gatekeeper.sh/v1
+kind: Assign
+metadata:
+  name: {{ printf "read-only-root-%s" $name | trunc 63 }}
```

Comment:
I'm thinking about adding this for all names so we can be sure that they aren't too long.

Reply:
Or should these have `"%s-read-only-root"` to keep things grouped based on the service? :thinking: